log/README.md

```diff
@@ -52,3 +52,10 @@ func main() {
 	stdlog.SetOutput(kitlog.NewStdlibAdapter(logger))
 }
 ```
+
+## Contributing
```

Comment:
Please add this section to the top-level README, rather than repeated in each package.